attributes/default.rb

```diff
@@ -140,8 +140,15 @@
 # default['openssh']['server']['chroot_directory'] = 'none'
 # default['openssh']['server']['banner'] = 'none'
 # default['openssh']['server']['subsystem'] = 'sftp /usr/libexec/sftp-server'
-default['openssh']['server']['trusted_user_c_a_keys'] = '/etc/ssh/ca_keys'
-default['openssh']['server']['revoked_keys'] = '/etc/ssh/revoked_keys'
+if platform_family?('windows')
```

Comment:
These can be refactored to use the return of the conditional for assignment rather than creating an intermediate variable. Also, drop the parentheses when there are no method arguments.

```ruby
default_ca_keys_path = if platform_family?('windows')
                         join_path(base_ssh_config_dir, 'ca_userkeys.pub')
                       else
                         '/etc/ssh/ca_keys'
                       end
```